lib/Kernel.php

```diff
@@ -45,7 +36,15 @@ public function getFactory() : Factory
      */
     public function getDirectives() : array
     {
-        return $this->directives;
+        return array_merge([
```

Comment:
This will re-instantiate all directives all the time: intentional?

Reply:
I am going to fix this in this PR. They don't need to be newly created every time.